target/linux/bcm27xx/patches-5.4/950-0089-cgroup-Disable-cgroup-memory-by-default.patch

```diff
@@ -17,16 +17,18 @@ Signed-off-by: Phil Elwell <phil@raspberrypi.org>
 
 --- a/kernel/cgroup/cgroup.c
 +++ b/kernel/cgroup/cgroup.c
-@@ -5722,6 +5722,8 @@ int __init cgroup_init_early(void)
+@@ -5721,6 +5721,10 @@ int __init cgroup_init_early(void)
+ 	return 0;
  }
  
- static u16 cgroup_disable_mask __initdata;
++static u16 cgroup_disable_mask __initdata;
```

Comment:
I am not sure if this patch still works as before, have a look at this change: 
https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/commit/?id=45e1ba40837ac2f6f4d4716bddb8d44bd7e4a251

```cgroup_disable_mask``` is not handled in the upstream code any more.

Reply:
I think  using this patch from kernel 5.10 is a better idea:
https://github.com/raspberrypi/linux/commit/2b13c54592135b6fab269517ed687fa9f80bf8e5